diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/Baidu.php b/src/Analyser/Header/Baidu.php
index 09607f86a..5daa15d9e 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/Baidu.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/Baidu.php
@@ -4,6 +4,8 @@
 
 class Baidu
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/BrowserId.php b/src/Analyser/Header/BrowserId.php
index 6b57b76c2..9e33223ce 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/BrowserId.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/BrowserId.php
@@ -9,6 +9,8 @@
 
 class BrowserId
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         if ($header == 'XMLHttpRequest') {
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/OperaMini.php b/src/Analyser/Header/OperaMini.php
index d8525bdf6..ab8981cc1 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/OperaMini.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/OperaMini.php
@@ -7,6 +7,8 @@
 
 class OperaMini
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/Puffin.php b/src/Analyser/Header/Puffin.php
index fdd34891b..d2a96d6be 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/Puffin.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/Puffin.php
@@ -6,6 +6,8 @@
 
 class Puffin
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/UCBrowserNew.php b/src/Analyser/Header/UCBrowserNew.php
index d3f20c72c..3ec856364 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/UCBrowserNew.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/UCBrowserNew.php
@@ -8,6 +8,8 @@
 
 class UCBrowserNew
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/UCBrowserOld.php b/src/Analyser/Header/UCBrowserOld.php
index 12caa9a1f..95b71bed7 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/UCBrowserOld.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/UCBrowserOld.php
@@ -7,6 +7,8 @@
 
 class UCBrowserOld
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/Useragent.php b/src/Analyser/Header/Useragent.php
index ebfa20010..d950f98f4 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/Useragent.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/Useragent.php
@@ -6,6 +6,10 @@ class Useragent
 {
     use Useragent\Os, Useragent\Device, Useragent\Browser, Useragent\Application, Useragent\Using, Useragent\Engine, Useragent\Bot;
 
+    private $data;
+
+    private $options;
+
     public function __construct($header, &$data, &$options)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Analyser/Header/Wap.php b/src/Analyser/Header/Wap.php
index 727feeeb7..ea5018a12 100644
--- a/vendor/whichbrowser/parser/src/Analyser/Header/Wap.php
+++ b/vendor/whichbrowser/parser/src/Analyser/Header/Wap.php
@@ -7,6 +7,8 @@
 
 class Wap
 {
+    private $data;
+
     public function __construct($header, &$data)
     {
         $this->data =& $data;
diff --git a/vendor/whichbrowser/parser/src/Model/Primitive/Base.php b/src/Model/Primitive/Base.php
index 352fd7d80..d29dd6829 100644
--- a/vendor/whichbrowser/parser/src/Model/Primitive/Base.php
+++ b/vendor/whichbrowser/parser/src/Model/Primitive/Base.php
@@ -6,6 +6,7 @@
  * @internal
  */
 
+#[\AllowDynamicProperties]
 class Base
 {
     /**

